distribution/openhabhome/configurations/openhab_default.cfg

```diff
@@ -2652,3 +2651,14 @@ tcp:refreshinterval=250
 #
 # Controller model, required, either '0BA7' for LOGO7 or '0BA8' for LOGO8
 #plclogo:bcu.model=0BA8
+###################################### KM200 Binding ##########################################
```

Comment:
There should be comments indicating which settings are required and which are optional.

If they're all required or they're all optional, a single note near the top stating this would suffice.